gno.land/pkg/gnoclient/signer_test.go

```diff
@@ -1 +1,3 @@
 package gnoclient
+
+// todo add signer tests
```

Comment:
Why add this comment?

Reply:
🤷 removed it:[46274e6](https://github.com/gnolang/gno/pull/1562/commits/46274e62e3f14eb7ba81484b84726470ed4e91d8)